task/dump-agent/src/main.rs

```diff
@@ -148,9 +108,15 @@ impl idl::InOrderDumpAgentImpl for ServerImpl {
         let sprot = drv_sprot_api::SpRot::from(SPROT.get_task_id());
         let mut buf = [0u8; 4];
 
+        let area = self.dump_area(0)?;
+
+        if area.agent != humpty::DumpAgent::Task {
```

Comment:
When would this happen?  Is this if `jefe` happened to use the 0th dump area?

Reply:
Adding a comment to explain this -- and at the same time fixing the agent nomenclature which has made this more unclear than it should be.